modules/ossm-security-mtls.adoc

```diff
@@ -15,17 +15,16 @@ By default, {ProductName} is set to permissive mode, where the sidecars in {Prod
 [id="ossm-security-enabling-strict-mtls_{context}"]
 == Enabling strict mTLS across the mesh
 
-If your workloads do not communicate with services outside your mesh and communication will not be interrupted by only accepting encrypted connections, you can enable mTLS across your mesh quickly. Set `spec.istio.global.mtls.enabled` to `true` in your `ServiceMeshControlPlane` resource. The operator creates the required resources.
+If your workloads do not communicate with services outside your mesh and communication will not be interrupted by only accepting encrypted connections, you can enable mTLS across your mesh quickly. Set `spec.security.mtls.enabled` to `true` in your `ServiceMeshControlPlane` resource. The operator creates the required resources.
```

Comment:
`spec.security.dataPlane.mtls` (and `spec.security.controlPlane.mtls`?)

Reply:
apiVersion: maistra.io/v1 should be replaced with apiVersion: maistra.io/v2 in a 2.0 doc . And 2.0 mtls path is different from 1.x (See luksa's comment above.)